src/plugin.js

```diff
@@ -9,15 +9,14 @@ import { postprocess } from '#postprocess';
  * @typedef PluginOptions
  * @type {Object}
  * @property {boolean} development // not in use yet
- * @property {boolean} usePreflight - force preflights to be included/excluded
+ * @property {boolean} test // internal use only
```

Comment:
I'm wondering how intuitive it will be to set a `test` flag compared to something like `skipPreflight` or `noPreflight`. `test` flag does basically one thing here - remove preflights from the plugin. If you think it may do more than that then ignore this comment 😄

Reply:
Yes, for sure. Name could be improved. Your suggestions sound good 👍🏻

Reply:
[Added](https://github.com/warp-ds/drive/pull/106/commits/f615e6eafbbd1cd03c6fb9e850d1c0a3dd82b197) `skipPreflight` instead and refactored.